clouddriver-elasticsearch/src/main/java/com/netflix/spinnaker/clouddriver/elasticsearch/ops/BulkUpsertEntityTagsAtomicOperation.java

```diff
@@ -62,19 +63,23 @@ public BulkUpsertEntityTagsAtomicOperationResult operate(List priorOutputs) {
     mergeTags(bulkUpsertEntityTagsDescription);
 
     Date now = new Date();
-    getTask().updateStatus(BASE_PHASE, "Retrieving current entity tags");
 
-    Map<String, EntityTags> existingTags = retrieveExistingTags(entityTags);
+    Observable.from(entityTags).buffer(50).subscribe(tags -> {
```

Comment:
Feels like you could just use guava's `List.partition(entityTags, 50)` and skip the rx usage (_as it doesn't look like you're doing to calls in parallel_).